src/components/MovieCard/movieCard.jsx

```diff
@@ -1,27 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { Modal } from '../Modal';
+import { EditForm } from '../Forms/editForm';
+import { DeleteForm } from '../Forms/deleteForm';
 
 import './movieCard.scss';
 
-export const MovieCard = ({ movie }) => (
-  <div className="movie-card">
-    <img
-      className="movie-card__image"
-      src={movie.poster}
-      alt={movie.title}
-    />
-    <div className="movie-card__info">
-      <div className="movie-card__descr">
-        <h2 className="movie-card__title">{movie.title}</h2>
-        <p className="movie-card__genre">{movie.genre}</p>
+import menuIcon from '../../images/more.svg';
+import closeIcon from '../../images/cancel.svg';
+
+export const MovieCard = ({ movie }) => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+  const [isMenuBtnShown, setIsMenuBtnShown] = useState(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState(false);
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+
+  return (
+    <div
+      className="movie-card"
+      onMouseEnter={() => setIsMenuBtnShown(true)}
+      onMouseLeave={() => { setIsMenuBtnShown(false); setIsMenuOpen(false); }}
```

Comment:
also this is not an error, but it's quite difficult to read. It's better to move this kind of callbacks to separate functions:
```js
const onMouseLeave = () => {
  setIsMenuBtnShown(false);
  setIsMenuOpen(false);
}
```

Reply:
Fixed, thanks